frontend/src/i18n/locales/en-US.json

```diff
@@ -346,7 +346,8 @@
       "same-addr-as-wallet": "the same address as the connected wallet",
```

Comment:
*arbitrary comment location*

Hey @alexkeating, we updated the format of the gitcoin contributor endpoint to be `/addresses/:address/is-gitcoin-contributor`. Please update this PR at your convenience to reflect the change. Thanks!